lib/providers/transit.js

```diff
@@ -79,6 +79,19 @@ class TransitProvider {
         return response;
       });
   }
+
+  /**
+   * Get a unique ID for the given token and secret
+   *
+   * @param {String} token
+   * @param {Object} secret
+   * @param {Object} secret.key
+   * @param {Object} secret.ciphertext
+   * @return {String}
+   */
+  static getSecretID(token, secret) {
```

Comment:
This looks like caching... We shouldn't try to cache transit input/output in tokend. That should be the responsibility of the consumer in this case.